### PR TITLE
(Edit)Added SortCommandParserTest.Updated UserGuide

### PR DESCRIPTION
**Pull request by DarrenCsAcc**
sort command parser test added

### CODE REVIEW

**Comment by DarrenCsAcc** [comment]
I added remove in the user guide as well as updated the table of the user guide

**Comment by DarrenCsAcc** [comment]
Added sort in user guide. Edited filter as discussed(filter [as/APPLICATIONSTATUS]).

File: docs/UserGuide.md [hunk 1]
```diff
@@ -164,6 +164,30 @@ Exits TrackAScholar.
...
+
+Format: `remove [ACCEPTED/REJECTED]`
+
+* The user selects the applicant(s) with application status of either `ACCEPTED` or `REJECTED` to be removed from TrackAScholar. 
+
+Examples:
+*  `remove accepted` removes all entries with ACCEPTED as an application status from TrackAScholar.
+*  `remove rejected` removes all entries with REJECTED as an application status from TrackAScholar.
+
+### Removing applicants by status : `sort`
```
Comment (ShummyOwnzYou) [inline docs/UserGuide.md]: Header name doesn't correspond with command name.

File: src/test/java/seedu/address/logic/parser/SortCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.model.applicant.Applicant;
+
+public class SortCommandParserTest {
```
Comment (ShummyOwnzYou) [inline src/test/java/seedu/address/logic/parser/SortCommandParserTest.java]: Need to add test case for sort by status, newly added functionality.

**Review by ShummyOwnzYou — COMMENTED** [review]
LGTM, other than the 2 comments that I've mentioned.

File: docs/UserGuide.md (hunk 1 above)
Comment (DarrenCsAcc) [reply docs/UserGuide.md]: Alright i will resolve it and let you know when i am done!

File: src/test/java/seedu/address/logic/parser/SortCommandParserTest.java (hunk 2 above)
Comment (DarrenCsAcc) [reply src/test/java/seedu/address/logic/parser/SortCommandParserTest.java]: Same for this!

**Comment by benjytan45678** [comment]
LGTM, the test cases can be further improved in v1.4

**Pull request merged**